issueWork/UI/UX__issue/README.md

```diff
@@ -0,0 +1,27 @@
+# UI / UX issue  
+
+## Changes made:  
+
+> 1. Complete overhall of webpage layout based on previously received feedback. Essentially creating a clean and somewhat minimal style.
+ - ### Laws:  
+   - **Aesthetic-Usability**
+   - **Jakob's Law**
+   - **Hick's Law**
+> 2. Attached and created an accordin to the hero section's breif about me. Which when clicked expands to a larger more indepth about me section.  
+ - ### Laws:  
+   - **Law of Proximity**
+   - **Doherty Threshold (currently 300ms)**
+   - **Law of Common Region**
+
+> 3. Created a Projects section. Currently grouping three mock projects with title, description, and what poblem it solved. May introduce languages and technology used to create project. This is dependant on feedback received.  
+ -  ### Laws:    
+    - **Law of Similarity**
+    - **Tesler's Law**
+
+> 4. Though not completed the contact form with be a modal with only 3 inputs. 
+ - ### Law:  
+   - **Postel's Law**
```

Comment:
Interesting, how does Postel's law apply here? This is about handling variable input

Reply:
Postel's Law here would mainly be:
- Anticipate virtually anything in terms of input, access, and capability while providing a reliable and accessible interface.
- The more we can anticipate and plan for in design, the more resilient the design will be.
Accessibility will be the top importance here as who do know who will be reviewing your portfolio. 
Secondly the modal is in anticipation of when one might want to get in contact with me. But making it a modal I allow the viewer to contact me the second they wish too. All while not pulling them away from the content that caused them to do so. (ie: not scroll jacking them to a different section on the page)